packages/manager/modules/cloud-connect/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@ovh-ux/manager-cloud-connect",
+  "version": "0.0.0",
+  "private": true,
+  "description": "OVHcloud CloudConnect product",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/cloud-connect"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
+  "dependencies": {
+    "bootstrap": "^4.3.1",
+    "lodash": "^4.17.14",
+    "moment": "^2.24.0"
+  },
+  "peerDependencies": {
+    "@ovh-ux/manager-core": "^9.0.0 || ^10.0.0",
+    "@ovh-ux/ng-at-internet": "^5.0.1",
+    "@ovh-ux/ng-at-internet-ui-router-plugin": "^3.1.0",
+    "@ovh-ux/ng-ovh-cloud-universe-components": "^1.5.0",
+    "@uirouter/angularjs": "^1.0.23",
+    "angular": "^1.7.5",
+    "angular-translate": "^2.18.1",
+    "oclazyload": "^1.1.0",
+    "ovh-api-services": "^9.39.1"
```

Comment:
Is it needed by the module ? If not it can just be added as dependency for the app

Reply:
Yes it is used in module for couple of API's